TabPane插槽类型推断错误

### Describe the bug


`
    <NTabPane>
      <template #tab>
        <NEllipsis></NEllipsis>
      </template>
    </NTabPane>
`

上述代码编译时，提示类型错误：

`
error TS2339: Property 'tab' does not exist on type 'Readonly<{ default?: (() => VNode<RendererNode, RendererElement, { [key: string]: any; }>[]) | undefined; prefix?: (() => VNode<RendererNode, RendererElement, { ...; }>[]) | undefined; suffix?: (() => VNode<...>[]) | undefined; }>'.

`

### Steps to reproduce

把如下代码放到.vue文件中即可，编译即可看到报错：

`
    <NTabPane>
      <template #tab>
        <NEllipsis></NEllipsis>
      </template>
    </NTabPane>
`

### Link to minimal reproduction

参考上述重新步骤即可

### System Info

```Shell
System:
    OS: macOS 15.4.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 70.53 MB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 23.11.0 - /opt/homebrew/bin/node
    npm: 11.3.0 - /opt/homebrew/bin/npm
    pnpm: 10.10.0 - /opt/homebrew/bin/pnpm
  Browsers:
    Chrome: 136.0.7103.93
    Safari: 18.4
  npmPackages:
    naive-ui: ^2.41.0 => 2.41.0 
    vue: ^3.5.13 => 3.5.13
```

### Used Package Manager

pnpm

### Validations

- [x] Read the [Contributing Guidelines](https://github.com/tusen-ai/naive-ui/blob/main/CONTRIBUTING.md).
- [x] Read the [docs](https://www.naiveui.com/en-US/).
- [x] Check that there isn't [already an issue](https://github.com/tusen-ai/naive-ui/issues) that reports the same bug to avoid creating a duplicate.
- [x] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/tusen-ai/naive-ui/discussions).
- [x] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/CHANGELOG.en-US.md
+++ b/CHANGELOG.en-US.md
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 2.41.1
+
+### Fixes
+
+- `n-tabs` Fix the type exception of the tab slots in TabPane.
+
 ## 2.41.0
 
 `2025-01-05`

--- a/CHANGELOG.zh-CN.md
+++ b/CHANGELOG.zh-CN.md
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 2.41.1
+
+### Fixes
+
+- `n-tabs` 修复了TabPane中tab Slots的类型异常。
+
 ## 2.41.0
 
 `2025-01-05`

--- a/src/tabs/src/TabPane.tsx
+++ b/src/tabs/src/TabPane.tsx
@@ -41,6 +41,7 @@ export type TabPaneProps = ExtractPublicPropTypes<typeof tabPaneProps>
 
 export interface TabPaneSlots {
   default?: () => VNode[]
+  tab?: () => VNode[]
   prefix?: () => VNode[]
   suffix?: () => VNode[]
 }

